code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -144,6 +153,8 @@
 		temperature = reference.temperature
 
 	// Handles the burning of atoms.
+	if(temperature < FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
```

Comment:
```suggestion
	if(cold_fire?)
```

Reply:
Why not? Order of setting?

Reply:
no you're right i'll change this and the others too